apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.IntegrationTests/SimplePubSubTest.cs

```diff
@@ -41,13 +42,18 @@ public SimplePubSubTest(PubsubFixture fixture)
 
         private readonly PubsubFixture _fixture;
 
-        private async Task RunBulkMessaging(int messageCount, int minMessagesize, int maxMessageSize, int maxMessagesInFlight, int initialNackCount, TimeSpan? timeouts = null)
+        private async Task RunBulkMessaging(
+            int messageCount, int minMessageSize, int maxMessageSize, int maxMessagesInFlight, int initialNackCount,
+            TimeSpan? timeouts = null, int? cancelAfterRecvCount = null)
         {
+            // Force messages to be at least 4 byte long, so an int ID can be used.
```

Comment:
Nit: byte => bytes

Reply:
(And don't we control the arguments? Can't we just validate that minMessageSize is at least 4?)